src/app/fyle/split-expense/split-expense.page.ts

```
@@ -866,105 +868,112 @@ export class SplitExpensePage implements OnDestroy {
+        map((costCenters: CostCenter[]) =>
+          costCenters.map((costCenter) => ({
+            label: costCenter.name,
+            value: costCenter,
+          }))
+        )
+      );
+    }
+    this.isCorporateCardsEnabled$ = orgSettings$.pipe(
+      map(
+        (orgSettings) =>
+          orgSettings.corporate_credit_card_settings && orgSettings.corporate_credit_card_settings.enabled
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)